sdk-base/src/main/java/com/mapbox/maps/plugin/ConfigProperties.kt

```diff
@@ -24,6 +24,22 @@ enum class ScrollMode(val value: String) {
   HORIZONTAL_AND_VERTICAL("horizontal-and-vertical"),
 }
 
+/**
+ * The enum controls how the puck is oriented
+ *
+ * @param value String value of this property
+ */
+enum class PuckBearingSource(val value: String) {
```

Comment:
@Chaoba I see we could set puck bearing source with xml - is there a way to set it programatically (via public API)? If yes - I guess we need to update `api` files to make our kotlin binary compatibility happy.

Reply:
Yes, will rebase this PR and update it.